server/controllers/events/gitlab_request_parser_validator.go

```diff
@@ -84,15 +84,15 @@ func (d *DefaultGitlabRequestParserValidator) ParseAndValidate(r *http.Request,
 		// comment on a merge request or a commit.
 		var subset struct {
 			ObjectAttributes struct {
-				NoteableType string `json:"noteable_type"`
+				NotableType string `json:"notable_type"`
```

Comment:
are you sure the gitlab API does not have this "misspelled"? since this is parsin a request payload it comes from gielab that way, no?

Reply:
that sounds about right, let me check with gitlab sdk

Reply:
let's right, let me exclude that from the changelist
<img width="728" alt="image" src="https://github.com/runatlantis/atlantis/assets/1580956/c775c32a-9969-4895-a799-024f2403fbb0">